README.md

```diff
@@ -89,6 +90,52 @@ tse.download(symbols=["وبملت", "ولملت"], write_to_csv=True)
 
 </div>
 
+### دانلود سابقه معاملات حقیقی و حقوقی به صورت مجزا
+برای دانلود سابقه معاملات حقیقی و حقوقی برای تمامی نمادها  میتوان از تابع زیر استفاده کرد
+
+<div dir="ltr">
+
+```python
+from pytse_client.download import download_client_types_record  
```

Comment:
download is an internal module and should not be accessed directly, Instead please include `download_client_types_record` function in `pytse_client/__init__.py` and import it like:
```python
from pytse_client import download_client_types_record
```

Reply:
I've applied the requested changes. 
In addition, all examples are updated due to them.